packages/core/test/runStream.test.ts

```
@@ -0,0 +1,610 @@
+    const anthropicChunks = results.filter((r) => r.type === "anthropic");
+    expect(anthropicChunks).toHaveLength(2);
+  });
+});
+
+export async function convertAsyncIterableToArray<T>(iterable: AsyncIterable<T>): Promise<T[]> {
+  const result: T[] = [];
+  for await (const item of iterable) {
+    result.push(item);
+  }
+  return result;
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)